src/kernel/include/thd.h

```diff
@@ -17,6 +17,7 @@
 #include "retype_tbl.h"
 #include "tcap.h"
 #include "list.h"
+#include "per_cpu.h"
```

Comment:
If thd.h already included per_cpu.h, fpu.h don't have to include per_cpu.h again.

Reply:
It will raise the "dereference to incomplete type" error when compile.